base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py

```diff
@@ -37,6 +37,13 @@ def compare_nssdb_with_cs(class_instance, subsystem, cert_tag):
 
     # Load cert from CS
     cert = subsystem.get_cert_info(cert_tag)
+
+    # get cert info from NSS database
+    cert_nssdb = subsystem.get_nssdb_cert_info(cert_tag)
+
+    if cert_nssdb:
+        cert.update(cert_nssdb)
+
```

Comment:
I think the `cert_nssdb` variable name is confusing because here it's a dictionary object, then in line 54 it changes into binary cert data.